ember-core/shared/src/main/scala/org/http4s/ember/core/Encoder.scala

```diff
@@ -47,31 +47,24 @@ private[ember] object Encoder {
         .append(resp.status.renderString)
         .append(CRLF)
 
-      resp.headers
-        .get[`Content-Length`]
-        .foreach { _ =>
-          appliedContentLength = true
-        }
-
       // Apply each header followed by a CRLF
       resp.headers.foreach { h =>
         if (h.isNameValid) {
+          appliedContentLength |= h.name == `Content-Length`.name
```

Comment:
I do not recognize this operator, is this optimized over an if with assignment. If so great, but specialized mutability stuff I don't really know.

Reply:
Good point. Maybe I should write it like this.
```scala
appliedContentLength = appliedContentLength || h.name == `Content-Length`.name
```